CHANGELOG.md

```diff
@@ -2,10 +2,54 @@
 
 ## Table of Contents
 
+- [v0.2.0](#v020)
 - [v0.1.0](#v010)
 - [v0.1.0-rc2](#v010-rc2)
 - [v0.1.0-rc1](#v010-rc1)
 
+
+## v0.2.0
+
+API Version: v1alpha1
+
+### API changes
+
+Service APIs has been renamed to Gateway API.
+[#536](https://github.com/kubernetes-sigs/service-apis/issues/536).
+
```

Comment:
I think it would be good to note somewhere in these release notes that our Go API has changed in a number of places to more closely follow the API conventions. This involved transitioning some fields to use pointers.

Reply:
Expanded the previous point in `Misc` section:
https://github.com/kubernetes-sigs/gateway-api/pull/548/files#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4edR44-R47